project/Easy-Travel-BD/EasyTravel.Web/wwwroot/assets/css/main.css

```
@@ -0,0 +1,2065 @@
+  display: block;
+  position: relative;
+  overflow: hidden;
+}
+
+.pulsating-play-btn:before {
+  content: "";
+  position: absolute;
+  width: 120px;
+  height: 120px;
+  animation-delay: 0s;
+  animation: pulsate-play-btn 2s;
```

Biome (1.9.4)

[error] Unexpected shorthand property animation after animation-delay



(lint/suspicious/noShorthandPropertyOverrides)